aikau/src/test/resources/testApp/js/aikau/testing/mockservices/responseTemplates/SearchBox/site_docs_search.json

```diff
@@ -22,7 +22,7 @@
       },
       {
          "nodeRef": "workspace:\/\/SpacesStore\/5fa74ad3-9b5b-461b-9df5-de407f1f4fe7",
-         "name": "budget.xls",
+         "name": "Really_really_really_really_I_mean_like_really_really_long_ name_without_so_much_of_a_hint_of_a_break_anywhere_in_it_like_none_at_all_whatsoever_its_madness",
```

Comment:
You're quite right... although this didn't stop the unit test from failing before the fix :)

I will change it though